csharp/src/ProtocolBuffers/ByteString.cs

```diff
@@ -139,7 +139,7 @@ public static ByteString FromBase64(string bytes)
         /// are copied, so further modifications to the array will not
         /// be reflected in the returned ByteString.
         /// </summary>
-        public static ByteString CopyFrom(byte[] bytes)
+        public static ByteString CopyFrom(params byte[] bytes)

```

Comment:
Yup, SGTM. Rather than do this here (with more churn on this PR) are you happy for me to start collecting issues like this to be dealt with as part of the cleanup?